workers/data_refinery_workers/processors/no_op.py

```diff
@@ -34,11 +42,13 @@ def _no_op_processor_fn(job_context: Dict) -> Dict:
 
     # This is a NO-OP, but we make a ComputationalResult regardless.
     result = ComputationalResult()
-    result.command_executed = "" # No op!
+    result.commands.append("") # No op!
```

Comment:
I think this should actually have a command name, especially since once Rich's recent work goes out the NO_OP command is actually going to be more of a CONVERT_GENE_IDS command.

Reply:
Are you saying that this line should be changed to:
```
result.commands.append("CONVERT_GENE_IDS")
```

Reply:
Maybe? At the moment all the gene conversion work is still using NO_OP as the name, but it's becoming a misnomer because it's actually doing a decent amount of work.

Let's keep it as NO_OP for now, and then we can think about using a CONVERT_GENE_IDS name in a broader sense at some point.